crates/routes/src/images.rs

```diff
@@ -96,15 +94,10 @@ async fn upload(
   body: web::Payload,
   client: web::Data<ClientWithMiddleware>,
   context: web::Data<LemmyContext>,
+  // require login
+  local_user_view: LocalUserView,
 ) -> Result<HttpResponse, Error> {
   // TODO: check rate limit here
-  let jwt = req.cookie("jwt").ok_or(error::ErrorUnauthorized(
```

Comment:
I take it this cookie is now renamed to `auth` ?

Reply:
Yes and its read by the session middleware which puts LocalUserView in the request extensions. This http handler takes LocalUserView as param, so it can only be called if it was set by the session middleware, if the user is logged in.